docs/source/common/weights_loading.rst

```diff
@@ -6,30 +6,32 @@
 
 .. _weights_loading:
 
-##########################
-Saving and loading weights
-##########################
+##############################
+Saving and Loading Checkpoints
```

Comment:
if you call it checkpoints, this should also be reflected in the filename :)

Reply:
done! thanks :)